models/noise_generator.h

```diff
@@ -96,6 +96,10 @@ frequency double - Frequency of sine modulation in Hz
 
    To obtain comparable results for different values of dt, you must
    adapt std.
+ - As The noise generator provides a different current for each of its targets,
```

Comment:
p.s. and maybe just a check to ensure that current = 0 before t_start and after t_end